### ISSUE TITLE
Add logging/monitoring to detect topic processing silently missing new messages 

### ISSUE THREAD

**Issue by toneillbroad** [clingen-data-model/genegraph#400]
Marina posted a slack message on the website channel that an updated GCI curation was republished twice but the updated information was not showing on the website. I saw that neither the stage nor production genegraph processed the gene_validity messages beyond a certain partition offset, but that dev genegraph had.  

There were no significant exceptions in either the stage or prod cluster logs. http://status.confluent.com
did not report any outages in GCP East for the time period.

I don't know what the underlying cause of this is. For each topic being processed in genegraph, a new thread is started and that thread polls the topic for new messages every x number of seconds. I suspect that the poll is silently blocking when this happens.

Adding a log message just before each topic poll and a monitor to detect the log messages at certain intervals would likely be hard (due to the variable nature of the topics being consumed) is perhaps a naive but effective strategy. Open to better ideas here.

**Comment by theferrit32** [comment]
@toneillbroad could it be possible it's related to this: https://github.com/clingen-data-model/genegraph/issues/194. That particular report says the error happened when loading RXNORM data but the underlying error case I think could happen with any data source. If an exception is thrown at a particular time, it halts stream ingesting.

**Comment by larrybabb** [comment]
@toneillbroad @theferrit32 can we quickly assess this at today's standup to see where this belongs in our work queue?

**Comment by larrybabb** [comment]
@toneillbroad @theferrit32 let's be sure to figure out whether this is a MUST do and if so whether it goes in backlog or selected for development. We should do this at our next standup.

**Comment by toneillbroad** [comment]
I did some sleuthing this past Monday as I found that this happening again.  I was able to narrow down which production pod was affected and logged in, installed some networking software (with @sjahl help) and diagnosed that there were less network connections to confluent.cloud than was expected - indicating one may have had an exception and was subsequently closed.

 Further investigation of the logs on the pod found that there was an exception raised by kafka:
00:43:03.086 [Thread-12] INFO  o.a.k.clients.FetchSessionHandler  - [Consumer clientId=consumer-2, groupId=null] Error sending fetch request (sessionId=541150413, epoch=1546710) to node 6: {}.
org.apache.kafka.common.errors.DisconnectException: null

Discussed with @tnavatar the high priority of a fix on this issue, and he took the lead by implementing a solution to determine and log the occurrence of the problem.
